package.json

```diff
@@ -2,7 +2,10 @@
   "name": "treetracker-admin",
   "version": "1.2.1",
   "description": "This portion of the project is to process tree data. TreeTracker's Admin Panel Frontend and RESTful API built with loopback.",
-  "dependencies": {},
+  "dependencies": {
```

Comment:
@elfiyang16 Why do you install the router here? It supposes to be under client , right?